### ISSUE TITLE
Checking channel chugs for 10 minutes and times out

### ISSUE THREAD

**Issue by beefeater7** [axcore/tartube#9]
...leaving no result.
![channelcheck](https://user-images.githubusercontent.com/10468970/60400720-89f54480-9b67-11e9-93b2-f93f43d7f42f.PNG)

URL: https://www.youtube.com/channel/UC-lHJZR3Gqxm24_Vd_AJ5Yw

**Comment by axcore** [comment]
There is occasionally a communications failure between Tartube and youtube-dl. I haven't found the cause yet. It is not connected with PewDiePie; I have actually completed lots of test runs on that channel without problems.

**Comment by axcore** [comment]
The error message is being produced by youtube-dl; I think there's nothing the Tartube code can do about it. I have checked the youtube-dl command line options and I can't see any way to vary the length of the timeout.

I have added a 60-second timeout to the Tartube code, but it will only work when checking videos/channels/playlists. It doesn't work when actually downloading them, because youtube-dl doesn't announce that it is about to download the JSON file.
